source/decode.c

```diff
@@ -579,3 +703,7 @@ int aws_http_decoder_get_encoding_flags(const struct aws_http_decoder *decoder)
 size_t aws_http_decoder_get_content_length(const struct aws_http_decoder *decoder) {
     return decoder->content_length;
 }
+
+void aws_http_decoder_set_logging_id(struct aws_http_decoder *decoder, void *id) {
```

Comment:
AFAICT this function is never called. Is that intentional?

Reply:
Youuuuuu ARE RIGHT!
thanks
fixin it